Instruments/source/TupleSkimmer.cxx

```diff
@@ -446,7 +446,14 @@ class TupleSkimmer {
         }
 
 
-        if (!ntuple::MetFilters(full_event.metFilters).PassAll()) return false;
+        if (!ntuple::MetFilters(full_event.metFilters).Pass(ntuple::MetFilters::Filter::PrimaryVertex) ||
```

Comment:
Do not recreate `MetFilters` instance for each `Pass`. Also use `using` for `ntuple::MetFilters::Filter` to not repeat `ntuple::MetFilters` many times.